### PR TITLE
Add Template config property

### PR DESCRIPTION
**Pull request by reintroducing**
Small change to add the ability to define a template path in the config. I tested this locally and it worked so please do the same and let me know if any changes are necessary.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
+.DS_Store
+node_modules
```
Comment (sapegin) [inline .gitignore]: Things like `.DS_Store` (anything created by your OS, IDE, etc.) should be in your global .gitignore.

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "react-styleguidist",
   "description": "React components styleguide",
-  "version": "0.1.8",
+  "version": "0.1.9",
```
Comment (sapegin) [inline package.json]: Please don’t change version number and changelog (at least the version).

**Comment by sapegin** [comment]
Thanks! The feature itself is fine ;-)

File: .gitignore (hunk 1 above)
Comment (reintroducing) [inline .gitignore]: Good thought, I removed it and set up a global (not sure why I never did it before).

File: package.json (hunk 2 above)
Comment (reintroducing) [inline package.json]: No problem, I was wondering how you want to handle this, removing it.

**Comment by reintroducing** [comment]
@sapegin Updated with the requested changes.

**Comment by sapegin** [comment]
:+1:

**Pull request merged**